src/Arcus.WebApi.Logging/Arcus.WebApi.Logging.csproj

```
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>    
+    <Authors>Arcus</Authors>
+    <Company>Arcus</Company>
+    <RepositoryType>Git</RepositoryType>
+    <PackageTags>Azure;WebAPI;App Services;Web App;Web;API</PackageTags>
+    <Description>Provides capabilities to easily build Web APIs running in Azure.</Description>
```

Given this is the logging package we should provide a more suited description for it.

How about this?
```suggestion
    <Description>Provides capabilities for easily integrating logging when building Web APIs running in Azure.</Description>
```